build_overrides/p6.gni

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2020 Project CHIP Authors
```

Comment:
```suggestion
# Copyright (c) 2021 Project CHIP Authors
```
And similar in various other new files?

Reply:
Why is this resolved?  I don't see this resolved in the actual diff...

Reply:
And I see a bunch of other files with 2020 copyrights, for that matter.

Reply:
@bzbarsky-apple : Sorry that I missed to update some files.. Now updates all the files copyright to 2021